server-side/fill-in-the-blank/scatter-plot-colab/spearman-compare/init.js

```diff
@@ -0,0 +1,188 @@
+console.clear()
+// d3.select('body').selectAppend('div.tooltip.tooltip-hidden')
+
+
+// window.bertProcessedVocab = python_data.vocab
+// var tokenizer = new BertTokenizer()
+// tokenizer.load()
```

Comment:
remove commented-out code.

add license header to all source files

Reply:
removed my junk headers / added license headers